packages/terra-search-field/src/SearchField.jsx

```diff
@@ -86,10 +98,20 @@ class SearchField extends React.Component {
   }
 
   render() {
-    const { placeholder, searchDelay, minimumSearchTextLength, onSearch, onInvalidSearch, ...customProps } = this.props;
-    const searchFieldClassNames = cx([
-      'searchfield',
+    const {
+      placeholder,
+      searchDelay,
+      minimumSearchTextLength,
+      onSearch,
+      onInvalidSearch,
+      isDisabled,
+      sizing,
+      ...customProps
+    } = this.props;
+
+    const searchFieldClassNames = cx(['searchfield',
```

Comment:
Nit: move the searchfield string to next line.

Reply:
Addressed